chia/daemon/server.py

```diff
@@ -1263,50 +1263,59 @@ def launch_service(root_path: Path, service_command) -> Tuple[subprocess.Popen,
     return process, pid_path
 
 
-async def kill_process(
-    process: subprocess.Popen, root_path: Path, service_name: str, id: str, delay_before_kill: int = 15
+async def kill_processes(
+    processes: List[subprocess.Popen],
+    root_path: Path,
+    service_name: str,
+    id: str,
+    delay_before_kill: int = 15,
 ) -> bool:
     pid_path = pid_path_for_service(root_path, service_name, id)
 
     if sys.platform == "win32" or sys.platform == "cygwin":
         log.info("sending CTRL_BREAK_EVENT signal to %s", service_name)
-        # pylint: disable=E1101
-        kill(process.pid, signal.SIGBREAK)
 
+        for process in processes:
+            kill(process.pid, signal.SIGBREAK)
     else:
         log.info("sending term signal to %s", service_name)
-        process.terminate()
+        for process in processes:
+            process.terminate()
 
     count: float = 0
     while count < delay_before_kill:
-        if process.poll() is not None:
+        if all(process.poll() is not None for process in processes):
             break
         await asyncio.sleep(0.5)
         count += 0.5
     else:
-        process.kill()
+        for process in processes:
+            process.kill()
         log.info("sending kill signal to %s", service_name)
```

Comment:
i dont think we want the same print for each process

Reply:
I left it as a single messages since it printed the service name, not the process ID or other process unique text.  This certainly could be adjusted to include the PID and print for each process.  Should I add that additional info?

```suggestion
            log.info(f"sending kill signal to: {service_name} (PID {process.pid})")
```

Reply:
sorry this while else clause played tricks with my eyes its just one print per service we can leave it like it is